nestkernel/random_generators.h

```diff
@@ -115,6 +115,13 @@ class BaseRandomGenerator
    * @param N Maximum value that can be drawn.
    */
   virtual unsigned long ulrand( unsigned long N ) = 0;
+
+  /**
+   * @brief Uses the wrapped RNG engine to draw an unsigned long from a Poisson distribution with mean lam.
```

Comment:
```suggestion
   * @brief Uses the wrapped RNG engine to draw an unsigned long from a Poisson distribution with mean mu.
```

In my opinion, `mu` is a bit better than `lam` as variable name here. Applies to the occurrences below as well.